packages/ui-library/src/components/radio-group/index.ts

```diff
@@ -19,79 +19,80 @@ export class BlrRadioGroup extends LitElement {
   @property() readonly?: boolean;
   @property() checked?: boolean;
   @property() name?: string;
-  @property() size: InputSizesType = 'md';
+  @property() sizeVariant: InputSizesType = 'md';
+  @property() hasLegend?: boolean;
   @property() required?: boolean;
-  @property() onChange?: HTMLElement['oninput'];
-  @property() onBlur?: HTMLElement['blur'];
-  @property() onFocus?: HTMLElement['focus'];
+  @property() blrChange?: HTMLElement['oninput'];
+  @property() blrBlur?: HTMLElement['blur'];
+  @property() blrFocus?: HTMLElement['focus'];
   @property() hasError?: boolean;
   @property() errorIcon?: SizelessIconType;
-  @property() hasGroupLabel?: boolean;
   @property() options!: RadioOption[];
   @property() hasHint = true;
-  @property() groupHintIcon?: SizelessIconType;
+  @property() groupHintMessageIcon?: SizelessIconType;
   @property() groupErrorMessage?: string;
   @property() groupHintMessage?: string;
-  @property() groupErrorIcon?: SizelessIconType;
-  @property() groupLabel?: string;
-  @property() direction?: 'vertical' | 'horizontal';
+  @property() groupErrorMessageIcon?: SizelessIconType;
+  @property() legend?: string;
+  @property() direction: RadioGroupDirection = 'horizontal';
 
   @property() theme: ThemeType = 'Light';
 
   protected render() {
-    if (!this.size) {
+    if (!this.sizeVariant) {
       return null;
     }
     const dynamicStyles = this.theme === 'Light' ? [formLight, radioLight] : [formDark, radioDark];
 
     const legendClasses = classMap({
       'blr-legend': true,
-      [`${this.size}`]: this.size,
+      [`${this.sizeVariant}`]: this.sizeVariant,
       'error': this.hasError || false,
     });
 
     const legendWrapperClasses = classMap({
       'blr-legend-wrapper': true,
-      [`${this.size}`]: this.size,
+      [`${this.sizeVariant}`]: this.sizeVariant,
     });
 
     const classes = classMap({
-      [`${this.size}`]: this.size,
+      [`${this.sizeVariant}`]: this.sizeVariant,
```

Comment:
Right now we have a mixed implementation of [`${this.sizeVariant}`] and [this.sizeVariant]. Since we already get what we want from the second part, we should convert [`${this.sizeVariant}`] to [this.sizeVariant].

Reply:
done